packages/editor/src/plugins/serlo-template-plugins/toolbar-main/toolbar-main.tsx

```diff
@@ -18,15 +18,19 @@ import { SaveModal } from '@serlo/frontend/src/serlo-editor-integration/componen
 import { useState } from 'react'
 
 import { ClientOnlyPortal } from './client-only-portal'
+import { LeaveConfirmationRenderNull } from './leave-confirmation-render-null'
 import { entity } from '../common/common'
-import { useLeaveConfirm } from '@/helper/use-leave-confirm'
 
 interface ToolbarMainProps {
   changes?: StateTypeReturnType<(typeof entity)['changes']>
   licenseId?: StateTypeReturnType<(typeof entity)['licenseId']>
   showSubscriptionOptions?: boolean
 }
 
+const isNextApp = () => {
+  return Boolean(document.getElementById('__next'))
```

Comment:
Did we not remove the __next tag from root?

Or do we expect this id somewhere else trotzdem?

Reply:
I think this PR only changes the targets to something we set ourselves. `__next` get's injected by automatically, I'm not even sure if it can be removed.